src/pyze/cli/charge-history.py

```diff
@@ -54,17 +56,40 @@ def _format_charge_history(ch):
         dateutil.tz.tzlocal()
     )
 
-    end_date = dateutil.parser.parse(
-        ch['chargeEndDate']
-    ).astimezone(
-        dateutil.tz.tzlocal()
-    )
+    if 'chargeEndDate' in ch:
+        end_date = dateutil.parser.parse(
+            ch['chargeEndDate']
+        ).astimezone(
+            dateutil.tz.tzlocal()
+        )
+        end_date = end_date.strftime(DATE_FORMAT)
+    else:
+        end_date = ''
+
+    if 'chargeDuration' in ch:
+        chargeDuration = format_duration_minutes(ch['chargeDuration'])
+    else:
+        chargeDuration = ''
+    
+    # chargeStartInstantaneousPower seems to be missing for some charging sessions
+    
+    if 'chargeStartInstantaneousPower' in ch:
+        chargeStartInstantaneousPower = '{:.2f}'.format(ch['chargeStartInstantaneousPower'] / 1000)
+    else:
+        chargeStartInstantaneousPower = ''
+        
+    if 'chargeBatteryLevelRecovered' in ch:
+        chargeBatteryLevelRecovered = ch['chargeBatteryLevelRecovered']
+    else:
+        chargeBatteryLevelRecovered = ''
 
     return [
         start_date.strftime(DATE_FORMAT),
-        end_date.strftime(DATE_FORMAT),
-        format_duration_minutes(ch['chargeDuration']),
-        '{:.2f}'.format(ch['chargeStartInstantaneousPower'] / 1000),
-        ch['chargeBatteryLevelRecovered'],
+        end_date,
+        chargeDuration,
+        chargeStartInstantaneousPower,
+        chargeBatteryLevelRecovered,
+        ch['chargeStartBatteryLevel'],
```

Comment:
Just in case it turns out to not always be present...
```suggestion
        ch.get('chargeStartBatteryLevel'),
```